src/script/utils/environment.ts

```diff
@@ -13,6 +13,8 @@ export const env = {
   safeUrlFetcher:
     'https://pwabuilder-safe-url.azurewebsites.net/api/getsafeurl',
   webPackageGeneratorUrl: 'https://pwabuilder-web-platform.azurewebsites.net/',
+  webPackageGeneratorUrlForm:
```

Comment:
Can you change this to `webPackageGeneratorFormUrl`? I like to have "Url" as the suffix here.